mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -0,0 +1,498 @@
+            startDate=dictClimo['startDateClimo'],
+            endDate=dictClimo['endDateClimo'])
+
+        # Compute annual climatology
+        annualClimatology = climatology.compute_annual_climatology(
+                            ds, calendar)
+
+        horizontalVel = annualClimatology.avgNormalVelocity
+        verticalVel = annualClimatology.avgVertVelocityTop
+        # Convert to numpy
+        horizontalVel = horizontalVel.values
+        verticalVel = verticalVel.values
+        velArea = verticalVel * areaCell[:, np.newaxis]
+
+        # Create dictionary for MOC climatology (NB: need this form
+        # in order to convert it to xarray dataset later in the script)
+        mocDictClimo = {'depth': {'dims': ('nz'), 'data': refTopDepth}}
+        for region in regionNames:
+            print '   Compute {} MOC...'.format(region)
+            print '    Compute transport through region southern transect...'
+            if region == 'Global':
+                transportZ = np.zeros(nVertLevels)
+            else:
+                maxEdgesInTransect = dictRegion['maxEdgesInTransect{}'.format(region)]
+                transectEdgeGlobalIDs = dictRegion['transectEdgeGlobalIDs{}'.format(region)]
+                transectEdgeMaskSigns = dictRegion['transectEdgeMaskSigns{}'.format(region)]
+                transportZ = _compute_transport(maxEdgesInTransect,
+                                                transectEdgeGlobalIDs,
+                                                transectEdgeMaskSigns,
+                                                nVertLevels, dvEdge,
+                                                refLayerThickness,
+                                                horizontalVel)
+
+            regionCellMask = dictRegion['{}CellMask'.format(region)]
+            latBinSize = config.getExpression(sectionNameClimo,
+                                              'latBinSize{}'.format(region))
+            if region == 'Global':
+                latBins = np.arange(-90.0, 90.1, latBinSize)
+            else:
+                indRegion = dictRegion['ind{}'.format(region)]
+                latBins = latCell[indRegion]
+                latBins = np.arange(np.amin(latBins),
+                                    np.amax(latBins)+latBinSize,
+                                    latBinSize)
+            mocTop = _compute_moc(latBins, nVertLevels, latCell,
+                                  regionCellMask, transportZ, velArea)
+
+            # Store computed MOC to dictionary
+            mocDictClimo['lat{}'.format(region)] = {
+                'dims': ('nx{}'.format(region)), 'data': latBins}
+            mocDictClimo['moc{}'.format(region)] = {
+                'dims': ('nz', 'nx{}'.format(region)), 'data': mocTop}
+
+        # Save to file
+        print '   Save global and regional MOC to file...'
+        #dsmoc = xr.Dataset.from_dict(mocDictClimo)
+        #dsmoc.to_netcdf(outputFileClimo, 'w')
```

Comment:
@milenaveneziani, you could probably add the metadata to the dictionary as well (units, description, etc) but I'm not sure that would be clearer than how you're doing it below.